docs/pages/desktop-access/directory-sharing.mdx

```diff
@@ -0,0 +1,188 @@
+- A browser on your local machine that supports the File System Access API,
+  which Teleport uses for Directory Sharing. We support the latest versions of
+  Chromium-based browsers like Google Chrome, Brave, and Microsoft Edge. You can
+  see a full compatibility table in the [Mozilla Developer Network
+  documentation](https://developer.mozilla.org/en-US/docs/Web/API/File_System_Access_API#browser_compatibility).
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/4. Share a directory
+
+To share a directory, open the Teleport Web UI and begin a desktop session by
+navigating to the "Desktops" tab on the sidebar, finding a desktop you want to
+connect to, and clicking "CONNECT".
+
+Once the session starts, click the three-dot menu on the top-right of the screen
+and click "Share Directory":
+
+![The "Share Desktop" button](../../img/desktop-access/share.png)
+
+When you first share a directory, your browser will prompt you to allow it to
+make changes in the directory, and you will need to grant these permissions to
+enable directory sharing.
+
+After you grant permissions, you will see a new directory in File Explorer on
+the remote desktop. The directory will have the same name as the directory on
+your local machine, but with "on teleport" appended to the name. 
+
+You can only share a single directory at a time.
+
+## Step 2/4. Edit files in your directory
+
+You can easily share files—and file modifications—between your local machine and
+the remote desktop.  
+
+### Editing on the remote side
+
+When you create, edit, or delete a file in the shared directory on the remote
+side, your browser will apply those changes to the directory on your local
+machine. 
+
+If you move a file into the shared directory on the remote side, that
+file will appear on the local side as well. And if you move a file outside the
+directory on the remote side, the remote desktop will copy it, keeping it within
+the shared directory. 
+
+<Notice type="warning">
+
+Directory Sharing does not support moving files between subdirectories within
+the shared directory on the remote side.
+
+</Notice>
+
+### Editing on the local side
+
+You can create, edit, and delete files in the shared directory on your
+local machine, as well as move files between subdirectories. 
+
+To view your local changes on the remote side, you will need to either refresh
+or re-open the remote directory, or re-open any individual files you changed
+locally. When you do this, the remote desktop will request the current state of
+the directory from the Teleport Web UI.
+
+## Step 3/4. Disconnect
+
+To stop sharing a directory, end your remote desktop session. Navigate to the
+three-dot menu on the upper-right of the desktop session in your browser and
+click "Disconnect." 
+
+The next time you start a session on the remote desktop, the directory will no
+longer be shared. You will need to share the directory again to access its
+content.
+
+## Step 4/4. Disable Directory Sharing
```

Comment:
Nit: arguably a bit awkward to have this as a "step", but if this is what we do elsewhere then its not a big deal.

This is a helpful example, but I think we should also update the [RBAC](https://goteleport.com/docs/desktop-access/rbac/#clipboard-access) section of the docs (with that perhaps linking to here for a more detailed example).

Reply:
I wanted to make this a step-by-step guide, and for me, putting this before we explain what Directory Sharing is wouldn't make as much sense. 

I'll plan to add a section to the RBAC page this week on this branch.